sigopt/objects.py

```diff
@@ -284,3 +284,9 @@ class Token(ApiObject):
   client = Field(six.text_type)
   experiment = Field(six.text_type)
   user = Field(six.text_type)
+
+
+class AggregatedObservation(ApiObject):
+  assignments = Field(Assignments)
+  value = Field(float)
+  value_stddev = Field(float)
```

Comment:
No, if all the observations are failed, none of them are selected as a "best" observation, so the endpoint returns an empty list